design/deployment_design.md

```diff
@@ -0,0 +1,70 @@
+#  **Dockerless CLI -** deployment
+
+This document is the design document for the deployment of the Python CLI. 
+
+Currently, our Python CLI can be deployed in the following ways:
+
+1. Install the CLI via the pip Python package manager like so: `pip3 install conjur-client`. If installed in this way, Python **is** required on machine. If installed via pip, the user can then run the CLI like so `conjur-cli --help`
```

Comment:
> This package basically combines both the Conjur Python SDK and the CLI, right?

Yes. 
Regarding splitting into the logic into two repos, how much of a development investment do you think it would it be?